test/open_mic_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/joke'
+require './lib/user'
+require './lib/open_mic'
+require 'pry'
+
+class OpenMicTest < Minitest::Test
+
+	def setup
+		@open_mic = OpenMic.new({location: "Comedy Works",\
+ date: "11-20-18"})
+		@sal = User.new("Sal")
+		@ali = User.new("Ali")
+		@joke_1 = Joke.new(1, "Why did the strawberry cross the road?",\
+"Because his mother was in a jam.")
+		@joke_2 = Joke.new(2, "How do you keep a lion from charging?",\
+"Take away its credit cards.")
+	end
+
+	def test_openmic_exists
+		assert_instance_of OpenMic, @open_mic
+	end
+
+	def test_openmic_attributes
+		assert_equal @open_mic.location, "Comedy Works"
+		assert_equal @open_mic.date, "11-20-18"
+		assert_equal @open_mic.performers, []
+	end
+
+	def test_welcome
+		@open_mic.welcome(@sal)
+		@open_mic.welcome(@ali)
+		assert_equal @open_mic.performers.count, 2
```

Comment:
this isn't a very robust assertion-- this could be 2 of anything. You want it to be two users (specifically, sal and ali), but this assertion would pass if it was a string with two characters, a hash with two key/value pairs, an array of integers, etc. So if you know it should be an array of sal and ali, you should assert that.  `assert_equal [@sal, @ali], @open_mic.performers.count`